### ISSUE TITLE
signature or required key missing

### ISSUE THREAD

**Issue by andreasloe** [gordboy/rtl8812au-5.9.3.2#9]
I appreciate the work you guys put into this software. Unfortunately, I have an issue with this driver. I am running the newest ubuntu
`No LSB modules are available.`
`Distributor ID:	Ubuntu`
`Description:	Ubuntu 20.04.1 LTS`
`Release:	20.04`
`Codename:	focal`
and the installation process runs through
`> sudo make install`
`install -p -m 644 8812au.ko  /lib/modules/5.8.0-38-generic/kernel/drivers/net/wireless/`
`/sbin/depmod -a 5.8.0-38-generic`
I an also  sudo modprobe 8812au but I read in kern.log

`Jan 16 19:13:22 linux kernel: [ 1576.456187] 8812au: loading out-of-tree module taints kernel.`
`Jan 16 19:13:22 linux kernel: [ 1576.456701] 8812au: module verification failed: signature and/or required key missing - tainting kernel`
`Jan 16 19:13:22 linux kernel: [ 1576.460551] usbcore: registered new interface driver rtl8812au`
Also, if I use iwconfig no wireless shows. I am using an DWA 172, 
`lsusb
Bus 002 Device 003: ID 2001:3318 D-Link Corp. `

Do you happen to have an idea what is wrong here?

**Comment by gordboy** [comment]
A few things.

20.04.1 is not the latest ubuntu, 20.10 is.

Secondly, where did you get the 5.8.0-38-generic kernel? If it is not from a proper Ubuntu repo it will not build signed modules.

If I were you, I would wait a few weeks for 20.04.2 which will have the 5.8 kernel, and all will be fine. In the meantime, kernel 5.4 as supplied, works fine here.

And thirdly, are you running NetworkManager? And did you edit

`/etc/NetworkManager/NetworkManager.conf`

as it says in the README ?

Fourthly, is you device actually compatible with this driver? You can paste the output of `lsusb` if you like.

**Comment by andreasloe** [comment]
Thank you for your prompt reply! I just checked with my PC and did the usual update/upgrade/dist-upgrade. I still have the 5.0.8 kernel and I cannot tell you why it did not upgrade to the latest version (I am not using any nightly, though). I guess that I try to sort out this issue because it has nothing to do with this driver. 

Also, I checked another wifi module tonight and made it working. So I have to say that my incentives to work in this issue at the moment are pretty low. I think I can close this now and reopen it if I have installed the newer kernel.
